ajax/libs/slim.js/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "slim.js",
+  "filename": "slim.js",
+  "version": "2.6.1",
+  "description": "Superpower Web Components",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eavichay/slim.js.git"
+  },
+  "author": "Avichay Eyal <eavichay@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://github.com/eavichay/slim-web-components#readme",
```

Comment:
I think you would like to use http://slimjs.com/ as the homepage.
Then, you can use `"homepage": "http://slimjs.com/"` .